### PR TITLE
lsat: use package from aperture

### PR DESCRIPTION
**Pull request by guggero**
Related to lightninglabs/pool#221.
Depends on https://github.com/lightninglabs/aperture/pull/51.

Updates the Loop code to use aperture's `lsat` package instead of the duplicated local one.

#### Pull Request Checklist
- [X] Update `release_notes.md` if your PR contains major features, breaking changes or bugfixes

### CODE REVIEW

**Review by bhandras — APPROVED** [review]
Awesome, LGTM :japanese_castle:

File: release_notes.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ This file tracks release notes for the loop client.
 ## Next release
 
 #### New Features
+- If the payment for an LSAT fails, it is now automatically re-tried.
```
Comment (carlaKC) [inline release_notes.md:18]: 🔥

**Review by carlaKC — APPROVED** [review]
Great feature 😎

**Comment by guggero** [comment]
Merged the dependent PR and updated aperture to `v0.1.6-beta`.

**Review by carlaKC — APPROVED** [review]
Nice 🥇

**Pull request merged**